code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -322,7 +329,7 @@ var/const/MAX_ACTIVE_TIME = 400
 	return
 
 /obj/item/clothing/mask/facehugger/proc/Die()
-	if(stat == DEAD)
+	if(stat == DEAD || real = 0)
```

Comment:
Imagine being so useless one of the top coders from TG has to come over here and tell you your code is awful.